src/test/java/org/fundacionjala/coding/pablo/Movies/NewReleaseTest.java

```diff
@@ -0,0 +1,47 @@
+package org.fundacionjala.coding.pablo.Movies;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Administrator on 3/23/2017.
+ */
+public class NewReleaseTest {
+
+    /**
+     * Test for calculate rented days less than 3.
+     */
+
+    @Test
+    public void calculateRentedDaysLessThan3() {
+
+        //given
+        NewRelease newRelease = new NewRelease("TestNewRelease");
+
+        //when
+        final double actualResult = newRelease.calculateAmount(2);
+
+        //then
+        final double expectedResult = 6.0;
+        assertEquals(0, expectedResult, actualResult);
```

Comment:
mmmmm
the first argument is the expected result then the second is the actual result